modules/aerservBidAdapter.js

```diff
@@ -0,0 +1,137 @@
+import * as utils from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
+import { BANNER, VIDEO, NATIVE } from 'src/mediaTypes';
+
+const BIDDER_CODE = 'aerserv';
+
+const REQUIRED_PARAMS = ['plc'];
+
+const ENVIRONMENTS = {
+  dev: 'dev-ads.aerserv.com',
+  stage: 'staging-ads.aerserv.com',
+  prod: 'ads.aerserv.com',
+};
+
+const PATH = '/as/js/prebid/';
+
+const VERSION = '1.0.0';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [VIDEO],
+
+  /**
+   * @param {object} bid the bid to validate
+   * @returns {bool} true if valid, false otherwise
+   */
+  isBidRequestValid: function (bid) {
+    return utils.hasValidBidRequest(bid.params, REQUIRED_PARAMS, BIDDER_CODE);
+  },
+
+  /**
+   * @param {bidRequests[]} bidRequests the requests to build
+   * @returns {Array} requests
+   */
+  buildRequests: function (bidRequests) {
+    const referrer = utils.getTopWindowUrl();
+    const requests = [];
+
+    bidRequests.forEach(bid => {
+      const env = ENVIRONMENTS[bid.params['env']] || ENVIRONMENTS['prod'];
+      const parameterStr = buildQueryParameters(bid, {url: referrer});
+      const type = getAdTypeOfBid(bid);
+
+      requests.push({method: 'GET', url: `//${env}${PATH}${type}/${VERSION}?${parameterStr}`, bidRequest: bid});
+    });
+
+    return requests;
+  },
+
+  /**
+   * @param {*} response The server response
+   * @param request The original request
+   * @returns {Bid[]} A single bid wrapped in an array.
+   */
+  interpretResponse: function (response, request) {
+    const bidResponses = [];
+
+    if (typeof response !== 'object' || response.error) {
+      return bidResponses;
+    }
+
+    if (isValidResponse(response, request.bidRequest)) {
+      const bidResponse = {
+        requestId: request.bidRequest.bidId,
+        bidderCode: spec.code,
```

Comment:
You shouldn't set this. Adapter aliases won't work quite right if you do. We'll set it automatically in the `bidderFactory`.

Our docs are being updated now... sorry for the confusion.